install/kubernetes/cilium/README.md

```diff
@@ -138,7 +138,7 @@ contributors across the globe, there is almost always someone available to help.
 | enableIPv6Masquerade | bool | `true` | Enables masquerading of IPv6 traffic leaving the node from endpoints. |
 | enableK8sEventHandover | bool | `false` | Configures the use of the KVStore to optimize Kubernetes event handling by mirroring it into the KVstore for reduced overhead in large clusters. |
 | enableK8sTerminatingEndpoint | bool | `true` | Configure whether to enable auto detect of terminating state for endpoints in order to support graceful termination. |
-| enableXTSocketFallback | bool | `true` | Enables the fallback compatibility solution for when the xt_socket kernel module is missing and it is needed for the datapath L7 redirection to work properly. See documentation for details on when this can be disabled: http://docs.cilium.io/en/stable/install/system_requirements/#admin-kernel-version. |
+| enableXTSocketFallback | bool | `true` | Enables the fallback compatibility solution for when the xt_socket kernel module is missing and it is needed for the datapath L7 redirection to work properly. See documentation for details on when this can be disabled: https://docs.cilium.io/en/latest/operations/system_requirements/#linux-kernel. |
```

Comment:
`/latest/` version of docs.cilium.io is generated from the master branch, and doesn't represent the latest stable release, but instead it's just whatever the most recent docs available in the tree. `/stable/` points to the current newest release.

I could see arguments that maybe we should change this, but at least for now this is the case. In general we try to avoid pointing users towards the `/latest/` versions since the install documentation doesn't make a huge amount of sense, and it could also mention features that are not yet available. It'd be easy for users to follow this one link here and then explore the rest of the docs and end up installing an unsupported version.

With that in mind, please change these links to point to the stable docs:
```suggestion
| enableXTSocketFallback | bool | `true` | Enables the fallback compatibility solution for when the xt_socket kernel module is missing and it is needed for the datapath L7 redirection to work properly. See documentation for details on when this can be disabled: https://docs.cilium.io/en/stable/operations/system_requirements/#linux-kernel. |
```

Reply:
Clicked the wrong button, using this to post the review requested.

Reply:
thanks :100: